### ISSUE TITLE
P Tag is larger than every other element!

### ISSUE THREAD

**Issue by jamesbirrellgray** [CJSCommonPlatform/ngGovUk#51]
It seems the p tag is 1.19em which is larger that a-tags etc.....?
<img width="337" alt="screen shot 2016-04-06 at 11 16 18" src="https://cloud.githubusercontent.com/assets/16034882/14313378/39fb82bc-fbe9-11e5-8a8a-763e944ee0ce.png">

**Comment by vygis** [comment]
@jamesbirrellgray this seems to be a design decision...

![screen shot 2016-04-07 at 10 18 05](https://cloud.githubusercontent.com/assets/2743507/14346436/27d364f8-fcaa-11e5-8d04-86adffd16203.png)

It wouldn't be hard to change but then we should amend the `x`, `xs` etc classes accordingly and make sure we don't break stuff in the existing apps.

**Comment by jamesbirrellgray** [comment]
@vygis why is an a tag smaller by default - it makes no sense!

**Comment by jamesbirrellgray** [comment]
@vygis can we make the change either to the default text (make 19px) so it applies to all elements - span, a etc?

**Comment by vygis** [comment]
19px seems excessive, so I suppose it's best to remove the size rule for the default `<p>` and let it inherit from the body, and then update the smaller p classes accordingly. Let's get the new guys on this :)

**Comment by jamesbirrellgray** [comment]
@pabloibanezcom this is actually a strange one! the edge case is <li><a></a><p></p><li> - the console says that they are the same size, however the p tag is bigger - maybe play with in the ngGov docs to see if you can replicate

**Comment by pabloibanezcom** [comment]
@jamesbirrellgray OK, I'm going to see what's the issue

**Comment by jamesbirrellgray** [comment]
`<ul class="row list-unstyled">
    <li data-ng-repeat="service in index.serviceItems" class="col-sm-4 ng-scope">
      <a ui-sref="service.title" data-ng-bind="service.title" class="ng-binding">Prosecuting Cases</a>
      <p data-ng-bind="service.description" class="ng-binding">Prosecuting Cases</p>
    </li><li data-ng-repeat="service in index.serviceItems" class="col-sm-4 ng-scope">
      <a ui-sref="service.title" data-ng-bind="service.title" class="ng-binding">Defending Cases</a>
      <p data-ng-bind="service.description" class="ng-binding"></p>
    </li><li data-ng-repeat="service in index.serviceItems" class="col-sm-4 ng-scope">
      <a ui-sref="service.title" data-ng-bind="service.title" class="ng-binding">Defending Cases</a>
      <p data-ng-bind="service.description" class="ng-binding"></p>
    </li>
  </ul>`

**Comment by jamesbirrellgray** [comment]
`<ul class="row list-unstyled"><li><a>foo</a><p>bar</p><li></ul>`

**Comment by pabloibanezcom** [comment]
I think the problem is em transformation is applied twice for p. Is gets transformated first as it's inside a ul element (this also applied to a) and then as it's a p element (here is where is the difference with a or other elements).

One suggestion could be excluding p from general ul rule but I'm going to think about it.

If anyone wants to know a bit more about this feel free to take a look at this
http://www.sitepoint.com/css-font-sizing-tutorial/

**Comment by pabloibanezcom** [comment]
I'm still not 100% sure about the desired result. Must both a and p to be the same size in all conditions?
Are you sure you want to dicrease the font size of the p elements?

**Comment by vygis** [comment]
Maybe this would be a good opportunity to review sitewide typography with UX?

**Comment by pabloibanezcom** [comment]
Yes. If we dicrease the current p size by removing the rule the difference between lead parapraph and the standard one will be quite big in my opinion.

**Comment by jamesbirrellgray** [comment]
Lets review this tomorrow

**Comment by pabloibanezcom** [comment]
Hi James,

Any update about this? As soon as I know what's the desired sized for p I will create the commit.

**Comment by pabloibanezcom** [comment]
Finally I removed the font transformation for regular paragraph.

https://github.com/CJSCommonPlatform/ngGovUk/commit/235d68e00936dfb05a1e606d7762cdad3cc98232
